content/en/resources/keps/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+linktitle: Enhancements
+title: Kubernetes Enhancement Proposals (KEPs)
+description: |
+  List of Kubernetes enhancements
+type: docs
+menu:
+  main:
+    weight: 40
+---
+
+These data come from the [kubernetes/enhancements]
+repository on GitHub.
+
```

Comment:
```suggestion

To see the original Enhancement issue, click on the KEP number.  To see a summary of what the KEP does, click on the Title.

```

... this behavior is useful, but unintuitive, so adding text explaining it helps.